digitalpy/core/zmanager/impl/default_routing_worker.py

```diff
@@ -54,26 +67,35 @@ def initialize_tracing(self):
             self.factory.register_instance(
                 "tracingproviderinstance", self.tracing_provider
             )
-        except Exception as e:
-            pass
+        except Exception as ex:
+            raise ex
+
+    def send_response(self, response: Response, protocol):
+        self.formatter.serialize(response)
+        message = protocol.encode()+b","+response.get_values()
+        self.integration_manager_sock.send(message)
 
+    def send_error(self, exception: Exception):
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>D102 Missing docstring in public method